src/surfing_penguin/API/survey_api.py

```diff
@@ -160,6 +160,11 @@ class answer_survey(Resource):
     def post(self):
         try:
             if survey_functions.id_get_survey(api.payload['survey_id']):
+                now = survey_functions.id_get_survey(api.payload['survey_id'])
+                now_answers = api.payload['answers']
+                for i in now_answers:
```

Comment:
`i` is normally for numbers. In my opinion it would be better to replace `i` with `answer` here. Also the `now_answers` should be renamed as well.

Reply:
fixed